### DISCUSSION TITLE
Notifying users

### DISCUSSION THREAD

**Discussion by Sevinfolds** [navidrome/navidrome#3324] [Ideas]
An idea has appeared (is it possible to implement it) to make it possible to notify (in the settings, for example, make a checkbox to enable the function) users about the appearance of new releases (tracks, albums) as an option in the web interface, e-mail. Another idea is to subscribe, for example, to an artist, and when he has a new album, then the users subscribed to him will receive a notification. I think these features would be cool :)

**Comment by deluan** [comment]
That's a great idea, but it will have to wait until after rewriting the UI (next quarter or so)
